config/migrations/20200923144227-notule-cleanup2.sparql

```diff
@@ -0,0 +1,19 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX brc: <http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/>
+PREFIX type: <http://kanselarij.vo.data.gift/id/concept/document-type-codes/>
+
+DELETE {
+  GRAPH ?g {
+       ?zitting ext:algemeneNotulen ?notule.
+       ?notule ?p ?o.
+  }
+}
+WHERE {
+    GRAPH ?g {
+      ?zitting ext:algemeneNotulen ?notule.
+      FILTER NOT EXISTS {?notule ext:getekendeDocumentVersiesVoorNotulen ?doc }.
+      ?notule ?p ?o.
```

Comment:
Als in cleanup 1 de beide relaties verwijdert worden is het hier kwestie van de bestaande notule op te kuisen
`
?notule a ext:Notule ;
             ?p ?o.
OPTIONAL { ?s ?pp ?notule . }
`
Die optional is zelfs niet nodig, er zijn normaal geen relatie meer tussen andere objecten en Notule

Reply:
Die filter not exists lijkt niet nodig tenzij je twijfelt of er nog notules bestaat met die specifieke relatie ook al heb je die relatie verwijdert in de vorige query

Reply:
Zo ok?